dashboard/src/components/AppEdit/AppEdit.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
+
+import { RouterAction } from "react-router-redux";
+import { IServiceBinding } from "../../shared/ServiceBinding";
+import { IApp, IChartState, IChartVersion } from "../../shared/types";
+
+import DeploymentForm from "../../components/DeploymentForm";
+
+interface IAppEditProps {
```

Comment:
please cleanup unused props.

Reply:
oops sorry, did not see you were passing all of this into DeploymentForm, ignore.